protocol/0057-TRAN-transfers.md

```diff
@@ -87,6 +87,41 @@ To be able to dispatch rewards to reward pools of multiple markets pro rata to t
 - `reward metric asset` — (the settlement asset of all markets that will be in scope for the transfer)
 - `market scope` — a subset of markets in which parties are eligible to be rewarded from this transfer.
   - If the market scope is not defined / an empty list, it is taken as all the markets that settle in the reward metric asset.
+
+A party can further control their recurring transfer funding the reward pool by defining the entities which are within scope. Entities within scope can be either individual parties or [teams](./0083-RFPR-on_chain_referral_program.md#glossary). When scoping individuals, a subset of keys can be detailed, and when scoping teams a specific set of team ids can be detailed.
+
+To support entity scoping, the transaction include the following fields:
+
+- `entity scope` - mandatory enum which defines the entities within scope.
+  - `ENTITY_SCOPE_INDIVIDUALS` - the rewards must be distributed directly amongst eligible parties
+  - `ENTITY_SCOPE_TEAMS` - the rewards must be distributed amongst directly eligible teams (and then amongst team members)
+- `individual scope` - optional enum if the entity scope is `ENTITY_SCOPE_INDIVIDUALS` which defines the subset of individuals which are eligible to be rewarded.
+  - `INDIVIDUAL_SCOPE_ALL` - all parties on the network are within the scope of this reward
+  - `INDIVIDUAL_SCOPE_IN_TEAM` - all parties which are part of a team are within the scope of this reward
+  - `INDIVIDUAL_SCOPE_NOT_IN_TEAM` - all parties which are not part of a team are within the scope of this reward
+- `team scope` - optional list if the reward type is `ENTITY_SCOPE_TEAMS`, field allows the funder to define a list of team ids which are eligible to be rewarded from this transfer
+- `staking_requirement` - the required minimum number of tokens staked for a party to be considered eligible. Defaults to `0`.
+- `notional_time_weighted_average_position_requirement` - the required minimum notional time-weighted averaged position required for a party to be considered eligible. Defaults to `0`.
```

Comment:
this is in terms of quantum? assume that's covered elsewhere but if not, should be covered somewhere

Reply:
I don't think it necessarily needs to be in terms of quantum?

As part of the recurring transfer at the moment you have to specify an `asset_for_metric` in the recurring transfer so all the markets in the scope of your reward transfer will be using the same settlement asset anyway.

Notional still needed though because you could have BTC / ETH / DOGE markets all settled in USDC within scope.

Reply:
But this is the average position requirement for the reward though? I didn't think this was specific to any asset?